packages/std-client/package.json

```diff
@@ -42,6 +42,7 @@
     "@latticexyz/recs": "workspace:*",
     "@latticexyz/solecs": "workspace:*",
     "@latticexyz/store": "workspace:*",
+    "@latticexyz/store-cache": "workspace:^",
```

Comment:
what command are you using to add this? wondering if we should just make them all `^`

Reply:
this happens when i do `pnpm add @latticexyz/store-cache`